.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+- '6'
+- '7'
+- '8'
```

Comment:
I'm using async/await. I think that's only available in node 8. What'll happen for 6 & 7?